setup.py

```diff
@@ -126,8 +128,13 @@ def recursive_data_files(directory, install_directory):
 ]
 
 try:
-    os_release = open("/etc/os-release").read()
-    ubuntu = "ID=ubuntu" in os_release
+    ubuntu = bool(
```

Comment:
:thinking: I think I know why you were looking at the `os_release` functions in 3.11

Reply:
Yeah and as it turns out it's added in [3.10](https://docs.python.org/3/library/platform.html#platform.freedesktop_os_release), not 3.11 as I thought.

With 3.10 we can say:

```python
import platform
ubuntu = "ubuntu" in  platform.freedesktop_os_release().get("ID_LIKE", "").split() + [platform.freedesktop_os_release()["ID"]]
```

Likewise this dict contains everything we need for `os_release.py`, although we could also replace that with a dependency on [`distro`](https://github.com/python-distro/distro), which itself has no dependencies outside the standard library.